entrypoint

```diff
@@ -2,9 +2,11 @@
 
 set -e
 
-if ${FLASK_DEBUG}; then
-    talisker.gunicorn.gevent webapp.app:create_app\(\) --reload --log-level debug --timeout 9999 --worker-class gevent --bind $1
-else
-    talisker.gunicorn.gevent webapp.app:create_app\(\) --worker-class gevent --name talisker-`hostname`
+RUN_COMMAND="talisker.gunicorn.gevent webapp.app:create_app() --bind $1 --worker-class gevent --name talisker-`hostname`"
+
+if [ "${FLASK_DEBUG}" = true ]; then
```

Comment:
This doesn't match up with Flask documentation that says `FLASK_DEBUG=1`, I wonder if it will cause confusion - http://flask.pocoo.org/docs/0.12/quickstart/#debug-mode

Would it be better to check if the variable isn't empty?

Reply:
Good point. Maybe. But it seems intuitive to me that you should be able to set `FLASK_DEBUG=false` it would be set to false. And I feel we should err on the side of debug being off rather than on.

I'll check for 1 explicitly.